### PR TITLE
[CognitiveLanguage] Drop 2.7 support

### PR DESCRIPTION
**Pull request by annatisch**

### CODE REVIEW

File: sdk/cognitivelanguage/azure-ai-language-conversations/CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-## 1.0.0b3 (2022-04-12)
+## 1.0.0b3 (2022-04-14)
 
 ### Features Added
 * Entity resolutions
 * Extra features
 
 ### Breaking Changes
-Input and output structures are different from previous release.
+* Input and output structures are different from previous release.
```
Comment (johanste) [inline sdk/cognitivelanguage/azure-ai-language-conversations/CHANGELOG.md:10]: As a reader of this comment, I'm not sure what I should take away from it :).

**Review by johanste — APPROVED** [review]
...but please consider updating the Breaking Changes entry in the readme...

File: sdk/cognitivelanguage/azure-ai-language-conversations/setup.py [hunk 2]
```diff
@@ -62,16 +48,15 @@
     classifiers=[
         "Development Status :: 4 - Beta",
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3 :: Only',
```
Comment (xiangyan99) [inline sdk/cognitivelanguage/azure-ai-language-conversations/setup.py:51]: I see other packages use

"Programming Language :: Python :: 3 :: Only",
        "Programming Language :: Python :: 3",
        "Programming Language :: Python :: 3.6",
        "Programming Language :: Python :: 3.7",
        "Programming Language :: Python :: 3.8",
        "Programming Language :: Python :: 3.9",
        "Programming Language :: Python :: 3.10",

Maybe we want to be consitent?

Comment (annatisch) [reply sdk/cognitivelanguage/azure-ai-language-conversations/setup.py]: Thanks! Updated :)

**Pull request merged**